Documentation/docs/README.md

```diff
@@ -0,0 +1,18 @@
+# Sphinx Documentation
+
+This is the source of the [ITK docs](docs.itk.org) website.
+
+## Build environment
+
+To compile the document locally create a python virtual environment and install the required packages.
+
+For example in Linux / macOS:
+
+```
+python -m venv env
+source env/bin/activate
+pip install -r requirements.txt
+```
+
+Use `make html` in this directory to build the documentation.
+Open `_build/html/index.html` in your browser to inspect the result.
```

Comment:
You may want to include a section like this one:
https://github.com/slicer/slicer.org#regenerating-favicons

Reply:
I added a simplified version in *_static/icon/README.md*.